giraffe/src/utils/PlotEnv.ts

```diff
@@ -397,7 +397,7 @@ export class PlotEnv {
     }
   }
 
-  public resetDomains = (): void => {
+  public resetDomains(): void {
```

Comment:
defining it as an instance property (as was done before) makes it not part of the public prototype interface, which makes it impossible to `spyOn`